### DISCUSSION TITLE
90s Consumer Tape Example

### DISCUSSION THREAD

**Discussion by smfthe1199** [ntsc-rs/ntsc-rs#111] [Show and tell]
https://github.com/valadaptive/ntsc-rs/assets/153495179/5fd8cc6b-2ae8-4f20-ac61-145633b0a7d5

**Comment by Degamisu** [comment]
wow! this is the best one yet! the scanlines are curved the wrong way though. other than that, its realistic!

**Reply by smfthe1199** [reply]
Thanks for this! Speaking of the Scanlines, It's actually a shader I used in my 86Box setup! That's right, I use 86Box, and? I am using a shader where the CRT Set's scanlines are like this; **wrongly curved** _as you say_, now this is normal tho and is noticable in Old PCs' CRT Monitors, if you just stick up your camera. Thanks for this!
